content/download/weekly/index.adoc

```diff
@@ -0,0 +1,50 @@
+---
+layout: simplepage
+title: Weekly Release Line
+---
+:toc:
+
+The Jenkins _Weekly Release Line_ delivers bug fixes and new features rapidly to users and plugin developers who need them.
+These releases are generally delivered on a weekly cadence, hence the name.
+For more conservative users, there is also a 
+link:/download/lts/[Long-Term Support (LTS)] line.
+
+## Model
+
+The "weekly" release cadence is **aspirational**.
+Generally we target releasing Jenkins every week, but sometimes we do out-of-order releases.
+To get notifications about new releases, it is possible to subscribe to link:https://www.jenkins.io/changelog/rss.xml[releases RSS feed].
+
+Conditions for new releases of the _Weekly Release Line_:
+
+* **Regular releases** with all features, improvements and bug fixes integrated since the last release.
```

Comment:
```suggestion
* **Regular releases** with all features, improvements, and bug fixes integrated since the last release.
```

Personal preference. Sadly, the style guide does not mention Oxford comma.

Reply:
@MarkEWaite was not it you who also put +1 on the suggestion by @MRamonLeon to remove it? I am +1 for documenting the recommendations for that in the style guide

Reply:
The original had a duplicate "and"which was the major problem.

Reply:
Ah right